app/src/components/landing/Appsumo/Appsumo.tsx

```diff
@@ -293,6 +308,13 @@ const AppSumoModal: React.FC = () => {
           <RQButton className="appsumo-add-btn" type="link" onClick={addAppSumoCodeInput}>
             + Add more codes
           </RQButton>
```

Comment:
Lets disable the `Add more codes` button too when user try to enter more than 10 codes in one go